modules/core/src/main/java/org/apache/ignite/internal/client/impl/connection/GridClientNioTcpConnection.java

```diff
@@ -646,10 +675,16 @@ private void removePending(long reqId) {
     private GridClientAuthenticationRequest buildAuthRequest() {
         GridClientAuthenticationRequest req = new GridClientAuthenticationRequest();
 
+        long reqId = reqIdCntr.getAndIncrement();
+
+        req.requestId(reqId);
+
```

Comment:
It's redundant if `makeRequest` is used to send request.

Reply:
fixed